apps/mosaico/src/Article/Article.purs

```diff
@@ -147,6 +156,51 @@ render { props, state, setState } =
                 , foldMap articleTimestamps letteraArticle
                 ]
             }
+        , DOM.ul
+            { className: "mosaico-article__some-tags-group"
+            , children:
+              [ DOM.li 
+                  { className: "mosaico-article__some-tags-group--facebook"
```

Comment:
Thanks! Yes that was helpful.

Reply:
Very nice explanation of BEM, @ae-mo. 

Possibly obvious, but the order of the element and element--modifier classes matters:

```css
/*
.button--dark can't go here, as it will get overridden by the .button class below.

.button--dark {
  background-color: black;
  color: white;
}
*/

.button {
  padding: 1rem;
  margin: 0.5rem;
  font-size: 1.2rem;
  color: black;
  background-color: white;
}

/* Must come below the .button class */
.button--dark {
  background-color: black;
  color: white;
}
```

I created a [CodePen demo of this](https://codepen.io/danielrlc/pen/MWpBGbQ), mainly as a reminder to myself of how the CSS cascade works (code further down a file overrides code higher up).